frontend/prisma/schema.prisma

```diff
@@ -15,16 +15,46 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model User {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  email     String?  @unique
-  image     String?
-  role      Role     @default(USER)
+model Clubs {
+  cid         String         @id @default(uuid())
+  email       String         @unique
+  password    String
+  info        ClubInfo[]
+  events      Events[]
+  categories  ClubCategories[]
 }
 
-enum Role {
-  USER
-  ADMIN
+model Events {
+  eid         String   @id @default(uuid())
+  cid         String
+  clubs       Clubs    @relation(fields: [cid], references: [cid])
+  title       String
+  description String?
+  location    String
+  start_time  DateTime
+  end_time    DateTime
+  posted_time DateTime @default(now())
 }
+
+model Categories {
+  type        String         @id
+  clubs       ClubCategories[]
+}
+
+model ClubCategories {
+  cid         String
```

Comment:
would it be good if we have the id + uuid restriction thing for this?

Reply:
Like the @@unique thing below?